src/hooks/use-toggle.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { act, renderHook } from '@testing-library/react-hooks'
+
+import useToggle from './use-toggle'
+
+describe( 'useToggle()', () => {
+  it( 'should be initialized to false', () => {
+    const { result } = renderHook( () => useToggle( false ) )
+    expect( result.current.state ).toBe( false )
+  } )
+
+  it( 'should toggle to true', () => {
+    const { result } = renderHook( () => useToggle( false ) )
+
+    // eslint-disable-next-line no-void
+    void act( () => result.current.toggle() )
+    expect( result.current.state ).toBe( true )
```

Comment:
Your tests should **never** reach into internal state. This is because you may be testing how you know the component to work (white box) vs how it works to the end user (black box)

Reply:
Well `.current` is how I can get the state of the hook. The resultant hook has 2 properties `current` and `errors`
![image](https://user-images.githubusercontent.com/44710980/103161929-ebbf5a80-47ae-11eb-97e7-cb496089f80c.png)

I followed this example in docs https://react-hooks-testing-library.com/usage/basic-hooks#rendering